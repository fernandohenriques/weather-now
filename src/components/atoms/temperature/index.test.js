import React from 'react';
import { shallow } from 'enzyme';
import Temperature from './index';

describe('<Temperature />', () => {
  it('Snapshot testing', () => {
    const wrapper = shallow(<Temperature number={5} />);

    expect(wrapper).toMatchSnapshot();
  });

  it('Snapshot testing with featured mode', () => {
    const wrapper = shallow(<Temperature number={19} featured />);

    expect(wrapper).toMatchSnapshot();
  });

  it('Should render Temperature with tag H2 if prop featured is true', () => {
    const wrapper = shallow(<Temperature number={31} featured />);

    expect(wrapper.exists()).toBeTruthy();
    expect(wrapper.find('h2')).toHaveLength(1);
  });
});
