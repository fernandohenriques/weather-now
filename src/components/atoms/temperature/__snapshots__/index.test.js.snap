// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Temperature /> Snapshot testing 1`] = `
<div
  className="container"
>
  <h2
    className="blue"
  >
    5
    <div
      className="degree"
    >
      °
    </div>
  </h2>
</div>
`;

exports[`<Temperature /> Snapshot testing with featured mode 1`] = `
<div
  className="container"
>
  <h2
    className="orange"
  >
    19
    <div
      className="degree"
    >
      °
    </div>
  </h2>
</div>
`;
